code/modules/cargo/packs/imports.dm

```diff
@@ -0,0 +1,260 @@
+/**
+ * Imports category.
+ * This is for crates not intended for goodies, but also not intended for departmental orders.
+ * This allows us to have a few crates meant for deliberate purchase through cargo, and for cargo to have a few items
+ * they explicitly control. It also holds all of the black market material and contraband material, including items
+ * meant for purchase only through emagging the console.
+ */
+
+/datum/supply_pack/imports
+	group = "Imports"
+	crate_name = "emergency crate"
+	crate_type = /obj/structure/closet/crate/internals
+
+/datum/supply_pack/imports/foamforce
+	name = "Foam Force Crate"
+	desc = "Break out the big guns with eight Foam Force shotguns!"
+	cost = CARGO_CRATE_VALUE * 2
+	contains = list(/obj/item/gun/ballistic/shotgun/toy = 8)
+	crate_name = "foam force crate"
+
+/datum/supply_pack/imports/foamforce/bonus
+	name = "Foam Force Pistols Crate"
+	desc = "Psst.. hey bud... remember those old foam force pistols that got discontinued for being too cool? \
+		Well I got two of those right here with your name on em. I'll even throw in a spare mag for each, waddya say?"
+	contraband = TRUE
+	cost = CARGO_CRATE_VALUE * 3
+	contains = list(/obj/item/gun/ballistic/automatic/pistol/toy = 2,
+					/obj/item/ammo_box/magazine/toy/pistol = 2)
+	crate_name = "foam force crate"
+
+/datum/supply_pack/imports/meatmeatmeatmeat // MEAT MEAT MEAT MEAT
+	name = "MEAT MEAT MEAT MEAT MEAT"
+	desc = "MEAT MEAT MEAT MEAT MEAT MEAT"
+	cost = CARGO_CRATE_VALUE * 6
+	contains = list(/obj/item/storage/backpack/meat)
+	crate_name = "MEAT MEAT MEAT MEAT MEAT"
+	crate_type = /obj/structure/closet/crate/necropolis
```

Comment:
I am looking at the code already on the necropolis crate and resultingly have two questions.

1. Does this spawn locked?
2. Does it spawn mining loot if you use a skeleton key on it?

Reply:
It doesn't spawn locked, but I should probably check the later half. I think tendril chests have most of that code? Gimme a bit